sdks/python/src/opik/evaluation/tasks_scorer.py

```diff
@@ -96,6 +98,7 @@ def run(
     workers: int,
     nb_samples: Optional[int],
     verbose: int,
+    project_name: str,
```

Comment:
after updating the code in the `evaluator.py`, it will become `project_name: Optional[str]`